ui/app/AppLayouts/Profile/panels/ContactsListPanel.qml

```diff
@@ -21,8 +21,8 @@ Item {
     property var contactsStore
     property var contactsModel
     property int panelUsage: Constants.contactsPanelUsage.unknownPosition
-    property bool scrollbarOn: false
-    readonly property int contactsListHeight: ((contactsList.count * contactsList.itemAtIndex(0).implicitHeight)+title.height)
+    property bool scrollbarOn: ((contactsListHeight - 50) > (root.height-50))
```

Comment:
@alexandraB99 this magic "50" shows up in a few files. Maybe it makes sense to have a constant for it somewhere, eg. in Theme ?

Reply:
not really, it's really specific to this contact list, a few pixels margin so that it can be scrolled completely at the bottom and the last item to be fully visible. I can let's say create a new variable in ContactsListPanel to keep it there, but maybe it's too much in this case.